client/state/themes/selectors/get-theme-for-atomic-transfer-dialog.ts

```diff
@@ -0,0 +1,12 @@
+import 'calypso/state/themes/init';
+import { AppState } from 'calypso/types';
```

Comment:
This should be `import type`.
```suggestion
import type { AppState } from 'calypso/types';
```